### PR TITLE
OSD-6549 hive specific operations on backplane

### PR DESCRIPTION
**Pull request by feichashao**
Add permissions for backplane on hive clusters.

WIP:
- Need to rebase after https://github.com/openshift/managed-cluster-config/pull/697/ merged.

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
Can we move this into https://github.com/openshift/managed-cluster-config/tree/master/deploy/backplane/srep ?

File: deploy/backplane/srep-hive/00-srep.namespace.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
```
Comment (wanghaoran1988) [inline deploy/backplane/srep-hive/00-srep.namespace.yml:2]: We don't need this.

**Comment by feichashao** [comment]
@wanghaoran1988 PTAL when you have time. Tested manually in a staging hive.

**Comment by 2uasimojo** [comment]
Promotion MR: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/15872

**Pull request merged**